### ISSUE TITLE
Fix Pet classifier model loading in ai-enrichment-light (missing preprocessor_config.json)

### ISSUE THREAD

**Issue by mikesvoboda** [mikesvoboda/nemotron-v3-home-security-intelligence#4469]
## Problem

The Pet classifier model fails to load in the `ai-enrichment-light` container with the following error:

```
ERROR - Failed to load pet classifier: Unrecognized image processor in /models/pet-classifier.
        Should have a `image_processor_type` key in its preprocessor_config.json of config.json,
        or one of the following `model_type` keys in its config.json: [long list of supported types]
```

## Impact

* Pet/animal detections (dogs, cats) are not enriched with breed/type classification
* Pet-related security events lack detailed context

## Root Cause

The model at `/models/pet-classifier` is missing the required HuggingFace configuration files:

* `preprocessor_config.json` with `image_processor_type` key
* Or `config.json` with valid `model_type` key

This suggests the model was saved in a format incompatible with `AutoImageProcessor.from_pretrained()`.

## Files to Investigate

* Model directory: `${AI_MODELS_PATH}/model-zoo/pet-classifier/`
* Loading code: `/ai/enrichment-light/models/pet_classifier.py`
* Check what files exist in the model directory

### Step 1: Diagnose model format

```bash
ls -la /export/ai_models/model-zoo/pet-classifier/
cat /export/ai_models/model-zoo/pet-classifier/config.json
```

### Step 2: Fix based on model type

**If it's a custom PyTorch model:**

* Don't use `AutoImageProcessor.from_pretrained()`
* Use manual image preprocessing (resize, normalize, to_tensor)
* Load model with `torch.load()` directly

**If it's a HuggingFace model missing config:**
Create `preprocessor_config.json`:

```json
{
  "image_processor_type": "ViTImageProcessor",
  "do_resize": true,
  "size": {"height": 224, "width": 224},
  "do_normalize": true,
  "image_mean": [0.485, 0.456, 0.406],
  "image_std": [0.229, 0.224, 0.225]
}
```

**If model needs replacement:**

* Use a working HuggingFace pet classifier
* Example: `google/vit-base-patch16-224` fine-tuned on pets
* Or use CLIP zero-shot classification with pet breed labels

### Step 3: Update loading code

Modify `/ai/enrichment-light/models/pet_classifier.py` to handle the model format correctly.

## Verification

1. Rebuild container: `podman-compose -f docker-compose.prod.yml build --no-cache ai-enrichment-light`
2. Check logs: `podman logs <container> | grep -i pet`
3. Expected: `Pet classifier model loaded successfully` or no error
4. Health check: `curl http://localhost:8096/health` should show `pet_classifier: true`
5. Test with pet image to verify classification works

## Related

* Epic: NEM-1395 (AI Pipeline Enhancements)
* Config: `ENRICHMENT_PET_SERVICE=light`
* Model path in compose: `${AI_MODELS_PATH}/model-zoo/pet-classifier:/models/pet-classifier:ro`
